.devcontainer/onCreate-conda.sh

```diff
@@ -28,9 +28,6 @@ ln -sf "$PYODIDE_RECIPE_BUILD_DIR" packages/.build || echo "Note: Could not crea
 
 conda run -n pyodide-env --live-stream pip install -r requirements.txt
 
-# https://pyodide.org/en/stable/development/new-packages.html#prerequisites
-conda run -n pyodide-env --live-stream pip install -e ./pyodide-build
```

Comment:
Presumably we need to add `pyodide-build` to `requirements.txt` for this to make sense...

Reply:
Changed to `make pyodide_build`.